src/meta/processors/admin/AdminClient.cpp

```diff
@@ -81,22 +83,24 @@ folly::Future<Status> AdminClient::addPart(GraphSpaceID spaceId,
   req.space_id_ref() = spaceId;
   req.part_id_ref() = partId;
   req.as_learner_ref() = asLearner;
-  auto ret = getPeers(spaceId, partId);
-  if (!nebula::ok(ret)) {
-    LOG(INFO) << "Get peers failed: " << static_cast<int32_t>(nebula::error(ret));
+  auto partHosts = getPeers(spaceId, partId);
+  if (!nebula::ok(partHosts)) {
+    LOG(INFO) << "Get peers failed: "
+              << apache::thrift::util::enumNameSafe(nebula::error(partHosts));
     return Status::Error("Get peers failed");
   }
 
-  req.peers_ref() = std::move(nebula::value(ret));
-  return getResponse(
+  req.peers_ref() = std::move(nebula::value(partHosts));
+  return getResponseFromPart(
       Utils::getAdminAddrFromStoreAddr(host),
       std::move(req),
       [](auto client, auto request) { return client->future_addPart(request); },
```

Comment:
it's really necessary to add logs figuring out which host to send for this client. the same as other functions